src/ResourceManager/ContainerInstance/AzureRM.ContainerInstance.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.0'
+ModuleVersion = '0.1.1'
```

Comment:
@yolo3301 you can revert any change made to versions in this PR. We will handle the versioning before publishing the module